File-Formats/DocIO/Working-with-Tables.md

```diff
@@ -966,6 +966,139 @@ private void AlignCellContentForTable(WTable table, Syncfusion.DocIO.DLS.Horizon
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Tables/Align-text-within-a-table).
 
+## Insert Image to Table Cell
+
+You can insert an image into table cell by accessing the paragraph in that cell. The following code example illustrates how to insert an image into table cell.
```

Comment:
You can insert an image into the table cell by accessing the paragraph in that cell. The following code example illustrates how to insert an image into a table cell: